base/scripts/code.antiscum.cs

```diff
@@ -43,13 +43,13 @@
     return;
   }
 
-  if (%force) {
-    if (%timeLeft < 0) {
-      %timeLeft = 0;
-    }
-    Client::sendMessage(%cl, 1, "You have " @ %timeLeft @ " seconds to bring the flag home.~wshell_click.wav");
-    return;
-  }
+  //if (%force) {
```

Comment:
was this requested by players? i don't know if i agree with this. this informs people when they leave the safe zone and the timer starts. without it, players won't be aware when the timer is ticking.